### ISSUE TITLE
[BUG]: v14 Bulk actions not being passed an entityType

### ISSUE THREAD

**Issue by mattbrailsford** [umbraco/Umbraco-CMS#16698]
**Describe the bug**
When implementing a bulk action it expects a constructor argument of type `UmbEntityBulkActionArgs<>` which is defined as:

````
export interface UmbEntityBulkActionArgs<MetaArgsType extends MetaEntityBulkAction> {
    entityType: string;
    meta: MetaArgsType;
}
````

However in the bulk actions execute method, when I access `this.args` the `entityType` is never defined.

Looking at the code in these 2 locations:

https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/main/src/packages/core/collection/components/collection-selection-actions.element.ts#L61

https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/main/src/packages/core/collection/components/collection-selection-actions.element.ts#L9

We can see that an entity type is never passed to the bulk actions.

**To Reproduce**
1. Create a bulk action for an entity
2. In the `execute` method, try to access `this.args.entityType`
3. Execute the bulk action
4. See that `this.args.entityType` is undefined

**Expected behavior**
This should be consistent with entity actions where the args entityType is correctly passed so I expect this should be the same for bulk actions.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Brave (Chromium)

**Comment by leekelleher** [comment]
_6 months later..._ I have a fix for this... #17606

**Comment by iOvergaard** [comment]
Will be released in 15.1.
